src/DynamicFormControlGroup.vue

```diff
@@ -4,7 +4,7 @@
             <span v-if="helpText" class="icon-small" v-tooltip="helpText">
                     <help-circle-icon></help-circle-icon>
                 </span>
-            <span v-if="required" class="small">(required)</span>
+            <span v-if="required" :class="{'text-danger': isRed(controlGroup.controls[0].value), small: true}">(required)</span>
```

Comment:
I guess this should be `!!controlGroup.controls.find(c => isRed(c.value))` (is any control in the group missing, not just the first one)